src/proxy/Client.ts

```diff
@@ -53,6 +58,10 @@ class Client extends Base {
   getAllByQuery(readQueryInput: IReadQueryInput): Promise<IPagination<IClient>> {
     return this.getAllByQueryHelper<IClient>(readQueryInput);
   }
+
+  getAllPaginatedByQuery(readQueryInput: IReadQueryInput): Promise<IPagination<IClient>> {
```

Comment:
Each model will have to implement this method? even if it does nothing?
Imagine this will require us to update de module every time we need to use a new model, is that correct?